TransferList/TransferList.js

```diff
@@ -106,7 +105,7 @@ const TransferListBase = kind({
 		 * @default 'move'
 		 * @public
 		 */
-		firstListOperation: PropTypes.oneOf(['move', 'copy', 'delete']),
+		firstListOperation: PropTypes.oneOf(['move', 'copy']),
```

Comment:
Please add a comment in the PR description/resolution stating that you have removed the "delete" operation, why and how it can be done from now on (also add this as a comment on the ticket).